third-party/spamassassin/src/test/java/org/apache/james/spamassassin/SpamAssassinExtension.java

```diff
@@ -146,6 +157,10 @@ public void sync(String user) throws UnsupportedOperationException, IOException,
         public void dump(String user) throws UnsupportedOperationException, IOException, InterruptedException {
             spamAssassinContainer.execInContainer("sa-learn", "--dump", "magic", "-u", user);
         }
+
+        public void dumpAll() throws UnsupportedOperationException, IOException, InterruptedException {
```

Comment:
what dumpAll means ? I guess it's wiping data from spamassassin but that would be better to express it in less specific words